contracts/staking/IStaking.sol

```diff
@@ -265,17 +231,8 @@ interface IStaking {
         view
         returns (uint96);
 
-    /// @notice Returns staked amount of T, Keep and Nu for the specified
-    ///         staking provider.
-    /// @dev    All values are in T denomination
-    function stakes(address stakingProvider)
-        external
-        view
-        returns (
-            uint96 tStake,
-            uint96 keepInTStake,
-            uint96 nuInTStake
-        );
+    /// @notice Returns staked amount of T for the specified staking provider.
+    function tStake(address stakingProvider) external view returns (uint96);
```

Comment:
What about `stakeAmount`, or even just `stake`?

Reply:
`stake` is used, I'm fine with `stakeAmount`